packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -78,6 +79,7 @@ class CliServiceProvider implements ServiceProvider {
    * @param {string} uri - optional URI for telemetry.
    */
   constructor(mongoClient: MongoClient, uri?: string) {
+    super(mongodb);
```

Comment:
This is nice because we can modify the BSON the driver uses directly, without having to import the `bson` package.